src/screens/WebsocketResearchScreen/SignedChatScreen.tsx

```diff
@@ -42,6 +54,20 @@ const SignedChatScreen = () => {
     flatlistRef.current?.scrollToEnd();
   };
 
+  const moveChatToAnon = async () => {
+    try {
+      setLoading(true);
+      await moveToAnonymousChannel({
+        oldChannelId: selectedChannel?.id,
+        targetUserId: memberChat.user_id
+      });
+    } catch (e) {
+      console.log('error moving chat to signed channel', e);
+    } finally {
+      setLoading(false);
+    }
+  };
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There's a console.log statement on line 65 which should be replaced with a more robust logging mechanism or removed entirely if it was only used for debugging purposes. 

```diff
- console.log('error moving chat to signed channel', e);
+ // replace with your preferred logging method
+ logError('error moving chat to signed channel', e);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->